code/modules/uplink/uplink_item.dm

```diff
@@ -602,7 +603,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 			deflecting all ranged weapon fire, but you also refuse to use dishonorable ranged weaponry."
 	item = /obj/item/sleeping_carp_scroll
 	cost = 17
-	surplus = 0
+	surplus = 45 //less likely than the revolver so people dont see it in every other surplus crate
```

Comment:
Lord, no. The potential for Carp + X cheese is too high. /tg/ did it right removing the damn thing.